initialise/MirahezeFunctions.php

```diff
@@ -172,22 +163,19 @@ public static function readDbListFile( string $dblist, bool $onlyDBs = true, ?st
 				}
 			}
 
-			return $databasesArray['databases'][$database] ?? '';
+			return $databasesArray['databases'][ $database ] ?? '';
 		} else {
 			global $wgDatabaseClustersMaintenance;
-
 			$databases = $databasesArray['databases'] ?? [];
 
 			if ( $wgDatabaseClustersMaintenance ) {
-				$databases = array_filter( $databases, static function ( $data, $key ) {
-					global $wgDBname, $wgDatabaseClustersMaintenance;
-
+				$databases = array_filter( $databases, static function ( $data, $key ) use ( $wgDatabaseClustersMaintenance ) {
+					global $wgDBname;
```

Comment:
```suggestion
				$databases = array_filter( $databases, static function ( $data, $key ) {
					global $wgDBname;
					global $wgDatabaseClustersMaintenance;
```

(and remove the global that's above this as well)